pytest_repo_health/plugin.py

```diff
@@ -57,10 +62,10 @@ def pytest_configure(config):
 
         if repo_health_path is not None:
             config.args.append(os.path.abspath(repo_health_path))
+    elif config.getoption("dependencies_health"):
+        if repo_health_path is not None:
+            config.args.append(os.path.abspath(repo_health_path))
```

Comment:
These follwoing two lines are common in both if conditions so you can move them to this if condition `if any([config.getoption("repo_health"), config.getoption("dependencies_health")])`

```
if repo_health_path is not None:
            config.args.append(os.path.abspath(repo_health_path))
```

Reply:
done